0.data-download/scripts/data_loader.py

```diff
@@ -25,13 +36,23 @@ def load_data(data_directory, adult_or_pediatric = "all"):
     samp_vs_dep_ids = set(samp_ids) & set(sample_df["DepMap_ID"].tolist())
     
     # subset data to only matching IDs (samples in both dependency and sample data)
-    sample_df = sample_df.loc[sample_df["DepMap_ID"].isin(dep_vs_samp_ids)]
+    sample_df = sample_df.loc[sample_df["DepMap_ID"].isin(dep_vs_samp_ids)].reset_index(drop=True)
     dependency_df = dependency_df.loc[dependency_df["DepMap_ID"].isin(samp_vs_dep_ids)]
     
+    if adult_or_pediatric == "test":
+      test_df = pd.read_csv(testing_data_file, index_col=0
+                               ).reset_index(drop=True)
+      return test_df
+    
+    if adult_or_pediatric == "train":
+      train_df = pd.read_csv(training_data_file, index_col=0
+                                 ).reset_index(drop=True)
+      return train_df
```

Comment:
This logic looks good, but just noting that all of your code above will execute (loading, subsetting data) with no effect if you enter these if statements.

Also, you probably should create a new variable (setting `adult_or_pediatric` ="test" or "train" doesn't really make sense)

Reply:
the solution might be to create an entirely different data load function (`load_train_test_data()`) or something like that

```python
train_df, test_df = load_train_test_data(data_dir=PATH)
```